manifests/j/JTL-SoftwareGmbH/JTL-Wawi/1.7.8.1/JTL-SoftwareGmbH.JTL-Wawi.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: JTL-SoftwareGmbH.JTL-Wawi
+PackageVersion: 1.7.8.1
+PackageLocale: en-US
+Publisher: JTL-Software GmbH
+PackageName: JTL-Wawi
+License: FREE
+ShortDescription: JTL-Wawi Setup

```

```suggestion
ShortDescription: The free inventory management system noticeably simplifies your business processes. Transparent flow of goods, better overview, full control.
```
Short description must give a description of the package